invoice/src/main/resources/org/killbill/billing/invoice/ddl.sql

```diff
@@ -106,7 +106,7 @@ CREATE TABLE invoice_item_history (
     rate numeric(15,9) NULL,
     currency varchar(3) NOT NULL,
     linked_item_id varchar(36),
-    quantity int,
+    quantity decimal(18, 9),
```

Comment:
Shouldn't this be symmetric with subscriptions?

Reply:
That was my thinking as well, but as I was looking for what we **already have** (in invoice), I discovered our scheme is inconsistent: [invoice_items](https://github.com/killbill/killbill/blob/work-for-release-0.23.x/invoice/src/main/resources/org/killbill/billing/invoice/ddl.sql#L71) versus [invoice_items_history](https://github.com/killbill/killbill/blob/work-for-release-0.23.x/invoice/src/main/resources/org/killbill/billing/invoice/ddl.sql#L109)